### ISSUE TITLE
how to work with tensorboard?

### ISSUE THREAD

**Issue by li-yinan** [tensorflow/tfjs#686]
Is there any plan to support tf.summary?

**Comment by caisq** [comment]
cc @tafsiri @nkreeger

**Comment by nkreeger** [comment]
It's something we've been thinking about - but we don't have concrete plans. TFX announced a few exiting projects yesterday https://twitter.com/TensorFlow/status/1039244472050180096 and we'll probably sync with them + tensorboard integration for a TFJS solution.

For now, we recommend rolling your own graphs and we have some examples using vega-lite. For example, here in the iris training example: https://github.com/tensorflow/tfjs-examples/blob/master/iris/ui.js#L40

**Comment by caisq** [comment]
I just want to add that when https://github.com/tensorflow/tfjs/issues/631 lands, users will get a progress-bar logger when tf.Models are training in tfjs-node or tfjs-node-gpu, by default. Then users will be able to at least see the progress, the loss and metric values and measures of training speed in the console.

**Comment by caisq** [comment]
Re-opening this issue as we are working on enabling scalar-summary writing from tfjs-node to TensorBoard now.
